src/client.rs

```diff
@@ -181,20 +182,20 @@ impl<T: AsRef<str>> AuroraClient<T> {
             };
             let response = self.near_client.call(request).await.unwrap();
             match response.outcome_proof.outcome.status {
-                near_primitives::views::ExecutionStatusView::SuccessValue(result) => {
+                views::ExecutionStatusView::SuccessValue(result) => {
                     let result_bytes = base64::decode(result).unwrap();
                     let result = SubmitResult::try_from_slice(&result_bytes).unwrap();
```

Comment:
```suggestion
                    let result = SubmitResult::try_from_slice(&result).unwrap();
```

There's a bit of a "gotcha" here when you update the nearcore libraries. They changed the view object so that the [base64 decoding happens automatically](https://github.com/near/nearcore/blob/master/core/primitives/src/views.rs#L1213) during deserialization. So we need to drop the base64 decode on our side. 

It would be nice to have some tests to catch this in CI in the future.

Reply:
Good catch 👍🏻